src/queries/community-champions.js

```diff
@@ -0,0 +1,30 @@
+export const communityChampions = `
+    query CommunityChampions {
+        allStrapiCommunityChampions(
+            sort: { fields: [firstName, middleName, lastName] }
+        ) {
+            nodes {
+                id
```

Comment:
Are we going to use this field later on?

Reply:
whoopsies i copied and pasted my old query into here and forgot to take out the id and sorting when modifying it, thanks for catching that!